packages/playground/src/components/custom_toast.vue

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+import { createToast, type ToastOptions } from "mosha-vue-toastify";
+
+const typeToColor: { [key: string]: string } = {
+  success: "#1aa18f",
+  danger: "#FF5252",
+  info: "#1aa18f",
+};
+
+export function createCustomToast(content: string, type: "danger" | "default" | "info" | "success" | "warning") {
```

Comment:
can we make the type `enum`? it would be better to auto-complete and not have typos